packages/wagmi-core/src/actions/sendEmailOtp.ts

```diff
@@ -0,0 +1,32 @@
+import type { Config } from "wagmi";
+
+export type SendEmailOtpParameters = {
+  email: string;
+};
+
+export type SendEmailOtpReturnType = void;
+
+/**
+ * Phase 1 of email OTP auth — request an OTP to be sent to the provided email.
+ *
+ * @param {Config} config - The shared Wagmi/Alchemy config.
+ * @param {SendEmailOtpParameters} parameters - Parameters for the OTP request.
+ * @param {string} parameters.email - The user's email address to receive the OTP.
+ */
+export async function sendEmailOtp(
+  config: Config,
```

Comment:
will this `config` be extended w/ any alchemy-specific stuff? or will it just use the default type from wagmi?